packages/epo-react-lib/src/styles/globalStyles.ts

```diff
@@ -14,9 +14,10 @@ export const colorTokens: { [key: string]: string } = {
   neutral20: "#dce0e3",
   neutral30: "#d3d4d5",
   neutral40: "#9E9EA3",
-  neutral60: "#666666",
+  neutral60: "#6A6E6E",
```

Comment:
José advised that this token should be `#6a6e6e` instead of `#666666`, since rubin-obs-client still uses its own `globalStyles` this will only impact lib uses going forward.